charts/coturn/Chart.yaml

```
@@ -0,0 +1,14 @@
+apiVersion: v2
+name: coturn
+description: coturn - a STUN and TURN server
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.0.1
```

```suggestion
version: 0.0.42
```

(to be in inline with the other charts's versioning scheme - same version for all charts set by CI)

---

I've pushed a commit with this change just now, thanks for pointing it out.